### ISSUE TITLE
embed does not work

### ISSUE THREAD

**Issue by LennZone** [gamer2810/steam-miniprofile#1]
When i embed your code as given in your README, sadly nothing shows up.

**Comment by Crushedice** [comment]
true.  
Exception: 
` Access to XMLHttpRequest at 'https://steamcommunity.com/miniprofile/238158335.html' from origin 'https://OwnWebsite.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

**Comment by vdedek** [comment]
I think now you need to include your Steam WEB API key which is registered on your website. With help of links like this: https://api.steampowered.com/ISteamUser/GetPlayerSummaries/v2/?key=*****mykey*****&steamids=***mysteamid*****) you can get all the info you need and then style it. Here is what I refer to: https://steamapi.xpaw.me/#ISteamUser/GetPlayerSummaries.

Would be nice to have the script reworked including this. Sadly I am not a programmer. @gamer2810 @1653032

**Comment by gamer2810** [comment]
I was using [CORS Anywhere by RobW](https://github.com/Rob--W/cors-anywhere) to fetch the Steam Mini-profile.
Sadly the demo now requires you to get a temporary key every time you use it so it does not work anymore.

I'm building my own CORS proxy but there's no definitive date yet. 

In the mean time you can deploy your own CORS server and proxy the steamcommunity links in the [index.html](https://github.com/gamer2810/steam-miniprofile/blob/master/index.html).

Example: `https://<YOUR_OWN_PROXY>/steamcommunity.com/miniprofile/${accountId}.html`

**Comment by neonsn0w** [comment]
Now the mini profile shows up but there is no data?
![Screenshot 2022-09-05 173541](https://user-images.githubusercontent.com/85235959/188482812-2ce88d54-8cd6-4312-a9c9-d56846442b36.jpg)

**Comment by gamer2810** [comment]
> Now the mini profile shows up but there is no data? ![Screenshot 2022-09-05 173541](https://user-images.githubusercontent.com/85235959/188482812-2ce88d54-8cd6-4312-a9c9-d56846442b36.jpg)

@neonsn0w It's still working on my end. 
Can you check if your profile is visible to public? Maybe launching an incognito session and check your steam account link?
If it's not visible then it can't be displayed :D
![image](https://user-images.githubusercontent.com/45266477/188484162-ff91b21a-72e1-4461-83d8-5f9f42c68ab0.png)

**Comment by neonsn0w** [comment]
> > Now the mini profile shows up but there is no data? ![Screenshot 2022-09-05 173541](https://user-images.githubusercontent.com/85235959/188482812-2ce88d54-8cd6-4312-a9c9-d56846442b36.jpg)
> 
> @neonsn0w It's still working on my end. Can you check if your profile is visible to public? Maybe launching an incognito session and check your steam account link? If it's not visible then it can't be displayed :D
> 
> Also please make sure that you are using the **AccountID** instead of the **SteamID**. You can check it on [SteamDB](https://steamdb.info/calculator/)
> 
> ![image](https://user-images.githubusercontent.com/45266477/188484162-ff91b21a-72e1-4461-83d8-5f9f42c68ab0.png)

Oh god.

I was actually using the SteamID instead of the AccountID.
Now it works.
Sorry for bothering!

**Comment by gamer2810** [comment]
No problem :+1: 

Issue closed as resolved
